main.go

```diff
@@ -55,9 +55,6 @@ func main() {
 		w.Write(utils.StaticFile("./static/index.html"))
 	})
 
-	// Route for shortening the URL
-	router.HandleFunc("POST /", handlers.Main)
-
```

Comment:
I guess we can remove this route? This route isn't meant to be used by any external services nor internally.